Cargo.toml

```diff
@@ -80,6 +80,11 @@ zlib-ng = ["flate2/zlib-ng"]
 rustls = ["crates_io_api/rustls", "reqwest/rustls-tls"]
 native-tls = ["reqwest/native-tls"]
 
+fancy-no-backtrace = ["miette/fancy-no-backtrace"]
+fancy-with-backtrace = ["fancy-no-backtrace", "miette/fancy"]
+
+log_release_max_level_info = ["log/release_max_level_info"]
```

Comment:
Doesn't that make it impossible to use `--log-level debug` in release builds?

Reply:
It is not enabled by default, but I want to make it a feature so people can enable it if they really don't need debug output.